CMakeLists.txt

```diff
@@ -54,6 +54,9 @@ option(TRITON_ENABLE_TESTS "Include tests in build" OFF)
 option(TRITON_ENABLE_GPU "Enable GPU support in libraries" OFF)
 option(TRITON_ENABLE_ZLIB "Include ZLIB library in build" ON)
 
+# Package Perf Analyzer with the Python Client -- Intended to be used by PA repo
+option(TRITON_PACKAGE_PERF_ANALYZER "Include Perf Analyzer in pip wheel" OFF)
```

Comment:
It looks like setting this to OFF contradicts the comment about packaging it? Also, does this mean that long-term, PA will be built in this repo? Or is this a temporary solution?

Reply:
It should be set to off by default, because, if left on, the python client may look for a PA binary that doesn't exist. The intention is that this flag be used by a PA repo build, because, from that perspective, we can guarantee that the binary will exist by the time we build the python client.

Long-term PA will be built in the PA repo. The next step after migration will be to remove PA from client. Regarding packaging the PA binary with the python client wheel, I think there needs to be more cross-team discussion given it introduces this strange dependency issue.